src/components/inputs/TextInput.tsx

```diff
@@ -1,17 +1,41 @@
-import React from 'react'
-import { TextInput as NativeInput } from 'react-native'
+import React, { useState } from 'react'
+import { TextInput as NativeInput, View, KeyboardTypeOptions } from 'react-native'
+import EyeIcon from '../../assets/icons/eye.svg'
```

Comment:
Can we at all look at using `@` shorthand syntax here like we used to do with webpack is there a way to add this to allow us to easily make sure we're all loading from the base directory that way we dont get lots of `../../../` in our components

Reply:
I've set up the `@/*` aliasing now and updated everywhere using back paths to use the new alias.